carbonmark-data/app/[locale]/details/retirement-trends-by-token/page.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
```

Comment:
everything needs to be a client component? why?

Reply:
Every time we use the <Trans/> Component, we need to use Client Components. Nevertheless thanks to your comment I found a workaround by using the t macro instead whenever possible. I initialy assumed that the t macro had the same drawback.